packages/rule-no-broken-links/README.md

```diff
@@ -0,0 +1,65 @@
+# Check for broken links (`@sonarwhal/rule-broken-links`)
+
+This rule checks and reports if any links in your page are broken.
+This includes anchor tag `href` value and image `src` attribute value.
+
+## Why is this important?
+
+Broken links gives your user a bad user experience.
+
+## What does the rule check?
+
+This rule finds all the anchor tags and image tags in your page and checks
+the `href` or the `src` attribute value is valid by issuing a request to
+the URL. If the response status is either `404` or `410` or `500` or `503`,
+ the url will be flagged as a broken link
```

Comment:
the is a white space at the beginning of the line

Reply:
Shouldn't there be a whitespace after comma ?

Reply:
If I'm not wrong it doesn't apply when there's a line break and md should render it correctly.

Reply:
Yep, md should render it correctly.